light/helpers_test.go

```diff
@@ -21,10 +21,14 @@ import (
 type privKeys []crypto.PrivKey
 
 // genPrivKeys produces an array of private keys to generate commits.
-func genPrivKeys(n int) privKeys {
+func genPrivKeys(n int, keyType crypto.KeyType) privKeys {
 	res := make(privKeys, n)
 	for i := range res {
-		res[i] = ed25519.GenPrivKey()
+		if keyType == crypto.BLS12381 {
```

Comment:
I know these are the two supported types, but maybe we want to default to edwards or error when the type doesn't match? Otherwise, we'll get a nil private key.

Reply:
I actually noticed we were generating BLS keys for both key types. Fixed it and also defaulting to edwards.